include_core/omrport.h

```diff
@@ -284,6 +284,55 @@
 #define OMRPORT_TIME_US_PER_SEC ((uint64_t) 1000000) /* microseconds per second */
 /** @} */
 
+/**
+ * @name Shared Semaphore
+ * Flags used to indicate type of operation for omrshsem_post/omrshsem_wait
+ * @{
+ */
+#define OMRPORT_SHSEM_MODE_DEFAULT ((uintptr_t) 0)
+#define OMRPORT_SHSEM_MODE_UNDO ((uintptr_t) 1)
+#define OMRPORT_SHSEM_MODE_NOWAIT ((uintptr_t) 2)
+/** @} */
+
+/**
+ * @name Shared Semaphore Success flags
+ * @anchor PortSharedSemaphoreSuccessFlags
+ * Success codes related to shared semaphore  operations.
+ * @{
+ * @internal OMRPORT_INFO_SHSEM* range from at 100 to 109 to avoid overlap
+ */
+#define OMRPORT_INFO_SHSEM_BASE 100
+#define OMRPORT_INFO_SHSEM_CREATED (OMRPORT_INFO_SHSEM_BASE)
+#define OMRPORT_INFO_SHSEM_OPENED (OMRPORT_INFO_SHSEM_BASE+1)
+#define OMRPORT_INFO_SHSEM_OPEN_UNLINKED (OMRPORT_INFO_SHSEM_BASE+2)
+#define OMRPORT_INFO_SHSEM_OPENED_STALE (OMRPORT_INFO_SHSEM_BASE+3)
+#define OMRPORT_INFO_SHSEM_PARTIAL (OMRPORT_INFO_SHSEM_BASE+4)
+#define OMRPORT_INFO_SHSEM_STAT_PASSED (OMRPORT_INFO_SHSEM_BASE+5)
+
+/** @} */
+
+#define OMRSH_MAXPATH EsMaxPath
+
+#define OMRSH_SEMAPHORE_ID "_semaphore_"
+
+#define OMRSH_DIRPERM_ABSENT ((uintptr_t)-2)
+#define OMRSH_DIRPERM (0777)
+#define OMRSH_PARENTDIRPERM (01777)
+#define OMRSH_DIRPERM_DEFAULT (0000)
+#define OMRSH_DIRPERM_DEFAULT_WITH_STICKYBIT (01000)
+#define OMRSH_BASEFILEPERM (0644)
+#define OMRSH_BASEFILEPERM_GROUP_RW_ACCESS (0664)
+
+#define OMRSH_SYSV_REGULAR_CONTROL_FILE 0
+#define OMRSH_SYSV_OLDER_CONTROL_FILE 1
+#define OMRSH_SYSV_OLDER_EMPTY_CONTROL_FILE 2
+
+/* Flags passed to "flag" argument of omrshsem_deprecated_open(). */
+#define OMRSHSEM_NO_FLAGS		0x0
+#define OMRSHSEM_OPEN_FOR_STATS		0x1
+#define OMRSHSEM_OPEN_FOR_DESTROY	0x2
+#define OMRSHSEM_OPEN_DO_NOT_CREATE     0x4
+
```

Comment:
For example, these constants should be aligned with spaces, or not aligned at all.

Reply:
Thank you for your comment. Modified in latest commit.